packages/pretty-format/src/index.js

```diff
@@ -191,14 +193,15 @@ function printMap(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDepth,
   return result + '}';
 }
 
-function printObject(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDepth, currentDepth, plugins, min, callToJSON, printFunctionName, escapeRegex, colors) {
+function printObject(val: Object, indent: string, prevIndent: string, spacing: string, edgeSpacing: string, refs: Refs, maxDepth: number, currentDepth: number, plugins: Plugins, min: boolean, callToJSON: boolean, printFunctionName: boolean, escapeRegex: boolean, colors: Colors): string {
   const constructor = min ? '' : (val.constructor ?  val.constructor.name + ' ' : 'Object ');
   let result = constructor + '{';
   let keys = Object.keys(val).sort();
   const symbols = getSymbols(val);
 
   if (symbols.length) {
     keys = keys
+      // $FlowFixMe string literal `symbol`. This value is not a valid `typeof` return value
       .filter(key => !(typeof key === 'symbol' || toString.call(key) === '[object Symbol]'))
```

Comment:
You mean that `.filter(key => toString.call(key) !== '[object Symbol]')` is enough?

I guess it is moving properties whose keys are symbols to follow the other.